<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doubanFM</class>
 <widget class="QWidget" name="doubanFM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>255</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>255</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>255</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#doubanFM{
	background-color:white;
}</string>
  </property>
  <widget class="QToolButton" name="pauseButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>0</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton#pauseButton{
	border:none;
	background-color: rgb(157, 214, 197);
}
QToolButton#pauseButton:hover{
	background-color:rgba(157,214,197,180);
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/pause.png</normaloff>:/icons/pause.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="userLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>32</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#userLabel{
	color:rgb(147,152,146);
}</string>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="albumImageLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>250</width>
     <height>255</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>255</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>255</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../icons.qrc">:/icons/use your imagination.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="artistLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ARTIST</string>
   </property>
  </widget>
  <widget class="QLabel" name="albumLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>75</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>&lt; Album &gt; YYYY</string>
   </property>
  </widget>
  <widget class="QLabel" name="songLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>115</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#songLabel{
	
	color: rgb(157, 214, 197);
}</string>
   </property>
   <property name="text">
    <string>song name</string>
   </property>
  </widget>
  <widget class="Phonon::SeekSlider" name="seekSlider" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>200</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal {
border: none;
height: 3px; /* the groove expands to the size of the slider by default. by giving it
a height, it has a fixed size */
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
margin: 0px 0;
}
QSlider::handle:horizontal {
/*background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f); */
background:rgb(157, 214, 197);
border: 0px solid rgb(157, 214, 197);
width: 0px;
margin: 0px 0; /* handle is placed by default on the contents rect of the groove.
Expand outside the groove */
border-radius: 0px;
}
QSlider::add-page:horizon{
	background:#c4c4c4;
}
QSlider::sub-page:horizon{
	background:rgb(157, 214, 197);
}</string>
   </property>
  </widget>
  <widget class="QWidget" name="volumnContainer" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>120</width>
     <height>15</height>
    </rect>
   </property>
   <widget class="QLabel" name="runningtime">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>40</width>
      <height>15</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>40</width>
      <height>15</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>15</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>60</width>
      <height>15</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>15</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>15</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider#volumeSlider::groove:horizontal {
border: none;
height: 3px; /* the groove expands to the size of the slider by default. by giving it
a height, it has a fixed size */
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
margin: 0px 0;
}
QSlider#volumeSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
border: 1px solid #5c5c5c;
width: 0px;
margin: 0px 0; /* handle is placed by default on the contents rect of the groove.
Expand outside the groove */
border-radius: 3px;
}
QSlider#volumeSlider::add-page:horizon{
	background:#c4c4c4;
}
QSlider#volumeSlider::sub-page:horizon{
	background:#5c5c5c;
}</string>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="volumeIcon">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>20</width>
      <height>15</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>15</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>15</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../icons.qrc">:/icons/volume.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="userLoginButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>195</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
	border:none;
	background-color: white;
}
QToolButton:hover{
	border:none;
	background-color:rgba(224, 226, 225,200);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/user.png</normaloff>:/icons/user.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="rateSongButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>195</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton#rateSongButton{
	border:none;
	background-color: white;
}
QToolButton#rateSongButton:hover{
	background-color:rgba(224, 226, 225,200);
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/heart.png</normaloff>:/icons/heart.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="byeSongButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>195</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Never Play</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton#byeSongButton{
	border:none;
	background-color: white;
}
QToolButton#byeSongButton:hover{
	background-color:rgba(224, 226, 225,200);
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/trash.png</normaloff>:/icons/trash.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QToolButton" name="nextSongButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>195</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Next Song</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton#nextSongButton{
	border:none;
	background-color: white;
}
QToolButton#nextSongButton:hover{
	background-color:rgba(224, 226, 225,200);
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
